### PR TITLE
vdr-addon: several fixes

### PR DESCRIPTION
**Pull request by CvH**
@mglae I added two updated plugins and some rather meh wirbelscan bump https://github.com/CvH/vdr-plugin-wirbelscan/commit/e1377b95312a138a41f93d7b06b3adb4ed4e7324 (not sure this is the way to go), the new sourcefile has dropped the Makefile and is only relying at the `build` script (can't get it working with it, didn't tried too long). Maybe its better to patch the makefile back in (bigger patch) - ofc proper fix would to make it work with the build script.
Would you have a look ?

I also added an changed patch, the "old" one is not applying due line-ending badness (windows line-endings at Wirbelscan but not at the patch).

### CODE REVIEW

**Comment by CvH** [comment]
@mglae `Sh.., the DOS line endings disappeared while creating the PR commit :-(`
yea we had this problem before, a lot linux tools like to convert mixed line endings files to linux line endings

**Comment by mglae** [comment]
Looks good (but not tested yet).

File: packages/addons/addon-depends/vdr-plugins/vdr-plugin-wirbelscan/patches/vdr-plugin-wirbelscan-01-fixes.patch [hunk 1]
```diff
@@ -1,27 +1,56 @@
...
+index 602e659..aafbec3 100644
+--- a/wirbelscan.c
++++ b/wirbelscan.c
+@@ -12,6 +12,7 @@
+ #include "menusetup.h"
+ #include "countries.h"
+ #include "satellites.h"
++extern TChannels NewChannels;
+ 
+ static const char *VERSION        = "2018.11.04";
```
Comment (mglae) [inline packages/addons/addon-depends/vdr-plugins/vdr-plugin-wirbelscan/patches/vdr-plugin-wirbelscan-01-fixes.patch]:
```
          APPLY PATCH (common)  packages/addons/addon-depends/vdr-plugins/vdr-plugin-wirbelscan/patches/vdr-plugin-wirbelscan-01-fixes.patch
patching file wirbelscan.c
Hunk #1 succeeded at 12 with fuzz 2.
```
Fuzz 2 means almost unpatchable. Version is still "2017.06.04".

**Comment by mglae** [comment]
1) Scanning test did succeed, functionality is OK and can be merged.

2) The build script relies on being executed from the the VDR tree. Getting it to work means writing a new one.

3) Instead of patching i suggest keeping a copy of Makfile.old from the 2016 package in our repo and add it to the $PACKAGE_BUILD path because of easier maintenance. I can do it, just tell me.

**Comment by belaid38** [comment]
@mglae  @CvH    plzz compile new vdr 109   i need test  i have WP2

**Comment by CvH** [comment]
@mglae removed patch and added Makefile, looks now less meh
```
  UNPACK   vdr-plugin-wirbelscan
      APPLY PATCH (common)   packages/addons/addon-depends/vdr-plugins/vdr-plugin-wirbelscan/patches/vdr-plugin-wirbelscan-01-fixes.patch
patching file wirbelscan.c
patching file common.c
  BUILD    vdr-plugin-wirbelscan (target)
```

@belaid38 building

**Pull request merged**